controllers/drpolicy_controller.go

```diff
@@ -243,6 +250,37 @@ func s3ProfileValidate(ctx context.Context, apiReader client.Reader,
 	return "", nil
 }
 
+func validateCIDRsFormat(ctx context.Context, drpolicy *ramen.DRPolicy, log logr.Logger) (string, error) {
```

Comment:
can this be `validateCIDR` and no need to pass the ctx as input as its not used anywhere?

Reply:
i have removed `ctx`